diagnostic_updater/diagnostic_updater/example.py

```diff
@@ -188,11 +190,10 @@ def check_upper_bound(stat):
 
     # You can broadcast a message in all the DiagnosticStatus if your node
     # is in a special state.
-    updater.broadcast(0, "Doing important initialization stuff.")
+    updater.broadcast(b'0', 'Doing important initialization stuff.')
 
-    pub1 = rospy.Publisher("topic1", std_msgs.msg.Bool, queue_size=10)
-    pub2_temp = rospy.Publisher("topic2", std_msgs.msg.Bool, queue_size=10)
```

Comment:
I don't know exactly what's the purpose of this `pub2_temp` publisher, but it seemed completely unused. Therefore, I went ahead and deleted it.
Same on the c++ API.